README.md

```diff
@@ -9,105 +9,48 @@
 
 # MagicScript Components ReactNative
 
+## Prerequisites
```

Comment:
we should mention that the user also needs Android Studio OR XCode as a prerequisite.  We can even link to React-Native setup instructions here.

Reply:
Link to `React Native CLI Quickstart` added.